#!/usr/bin/env python

PACKAGE = "gr_map_utils"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("distance_to_origin",  double_t,   0, "Distance to Origin Filter",  100.0, 10, 2000.0)

exit(gen.generate(PACKAGE, "gr_map_utils", "OSMMapConverter"))
